desk/app/metagrab.hoon

```diff
@@ -102,20 +102,27 @@
       ==
   ==
 ::
+++  urbit-agent  'urbit/1.0'
+++  chrome-agent  'chrome/123.0.0.0'
```

Comment:
If the second one is more likely to always work, why wouldn't we just hardcode-include it 100% of the time?

Reply:
I thought about this and thought you'd prefer this approach. I can revise to just always appear as chrome